src/app/containers/FrontPageSection/utilities/storySplitter.jsx

```diff
@@ -0,0 +1,41 @@
+import { splitEvery, splitAt, take } from 'ramda';
+
+export const getAllowedItems = (items, isFirstSection) =>
+  isFirstSection ? take(13, items) : take(10, items);
```

Comment:
Might be better to pull the values `10` and `13` out of this function and define them as constatants.

Reply:
Done.